Sources/ArcGISToolkit/Extensions/ArcGIS/AttachmentHelpers/AttachmentsFeatureElement.swift

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import Foundation
+
+/// Indicates how to display the attachments. If `list` is specified, attachments show as links. If `preview` is specified, attachments expand to the width of the view. Setting the value to `auto` allows applications to choose the most suitable default experience for their application.
```

Comment:
Splits into multiple lines and updates the doc for `auto` to match the case's doc.

```suggestion
/// Indicates how to display the attachments. 
/// If `list` is specified, attachments show as links.
/// If `preview` is specified, attachments expand to the width of the view.
/// Setting the value to `auto` allows applications to choose the most suitable default experience.
```

Also I think we might just be able to rely on the doc for each case to speak for itself. If so we could do:
```suggestion
/// Indicates how to display the attachments.
```